Makefile

```diff
@@ -16,3 +16,7 @@ install: graphex.cabal
 test: graphex.cabal
 	cabal test
 .PHONY: test
+
+fmt:
+	fd .hs --exec stylish-haskell -i
```

Comment:
I noticed it looked like you were using stylish-haskell (by the way the imports were aligned).

I added it to the shell + made this target. Didn't run it because it polluted the diff.